wellcomeml/ml/bert_semantic_equivalence.py

```diff
@@ -52,7 +52,7 @@ def _initialise_models(self):
             model_name = 'bert-base-cased'
             from_pt = False
         elif self.pretrained == 'scibert':
-            model_name = check_cache_and_download('scibert_scivocab_cased')
+            model_name = model_name = 'allenai/scibert_scivocab_cased'
```

Comment:
is this line `model_name = model_name = 'allenai/scibert_scivocab_cased'`?

Reply:
Yeah - I fixed that in a subsequent commit https://github.com/wellcometrust/WellcomeML/pull/43/commits/5f38110846cc2574535089921a976eff26652130